config.lua

```diff
@@ -39,6 +38,8 @@ vim.cmd([[
 
 -- Vimtex configuration.
 vim.g.vimtex_view_method = "zathura"
+-- Enable inverse search (set progpath to location of lvim executable)
+vim.g.vimtex_callback_progpath =  vim.fn.expand('$HOME/.local/bin/lvim')
 vim.g.vimtex_quickfix_enabled = 0
```

Comment:
I'm curious, why is this disabled by default for LunarVim? I would claim that the quickfix feature is very useful and quite well implemented in VimTeX. Is it because you are also loading texlab or something like that which gives the same functionality?

Reply:
That is a great question for @SingularisArt, who was the original author of the LaTeX `config.lua` for LunarVim. I would presume you're right -- LunarVim wants to support as many different users/packages as possible, and both texlab and VimTeX are being loaded in the config file.

Reply:
Cool; I _believe_ you still want to load the quickfix module even if you do use texlab. But you may want to disable the auto open feature. I.e., I would propose the following change:

```diff
-vim.g.vimtex_quickfix_enabled = 0
+vim.g.vimtex_quickfix_mode = 0
```